protocol/x/subaccounts/keeper/isolated_subaccount.go

```diff
@@ -2,16 +2,49 @@ package keeper
 
 import (
 	"math"
+	"math/big"
 
 	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/dydxprotocol/v4-chain/protocol/lib"
+	assettypes "github.com/dydxprotocol/v4-chain/protocol/x/assets/types"
 	perptypes "github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
 	"github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+type positionStateTransition uint
```

Comment:
Can we add some documentation on what each value of `positionStateTransition` means? It was not immediately clear to me what it means for `isolatedPerpetualStateTransition.transition` to be `opened` or `closed` at first glance